### PR TITLE
fix: add css and style loader to sass/stylus/less template

### PR DESCRIPTION
**Pull request by TheRealSyler**
**What kind of change does this PR introduce?**
Fixes the sass config generation.

**Did you add tests for your changes?**
No.

**If relevant, did you update the documentation?**
No, no change needed.

**Summary**

Related to #3171 

**Does this PR introduce a breaking change?**
No.

### CODE REVIEW

**Comment by alexander-akait** [comment]
Can you accept CLA?

**Comment by TheRealSyler** [comment]
> Can you accept CLA?

yes.

File: packages/generators/src/handlers/default.ts [hunk 1]
```diff
@@ -150,7 +150,7 @@ export async function questions(
       break;
   }
 
-  if (isCSS) {
+  if (isCSS || cssType === "SASS") {
```
Comment (alexander-akait) [inline packages/generators/src/handlers/default.ts:153]: I think we should generate it for `LESS` and `Stylus` too

Comment (TheRealSyler) [reply packages/generators/src/handlers/default.ts]: should work for all css types now.

**Review by alexander-akait — DISMISSED** [review]
/cc @webpack/cli-team Please review

**Review by rishabh3112 — APPROVED** [review]
Looks good.
Lets merge after CI is green.

**Review by rishabh3112 — CHANGES_REQUESTED** [review]
Looks like test for `init` in macos are taking longer.
Increasing timeout would be fine here.
Feel free to update it.

**Comment by TheRealSyler** [comment]
@rishabh3112 well see if it fixes the issue, i saw that one test failed because it ínstalled packages that weren't in the snapshot.

**Review by evenstensberg — APPROVED** [review]
Re-ran actions, and lgtm

**Comment by TheRealSyler** [comment]
I have increased the time again based on the time i saw in the test, but the limit is over 50min now, which might be bad but im not familiar enough with this project to judge that.

**Comment by rishabh3112** [comment]
> I have increased the time again based on the time i saw in the test, but the limit is over 50min now, which might be bad but im not familiar enough with this project to judge that.

It is bad but we can investigate it later. Feel free to create a issue for it. Thanks.

**Comment by rishabh3112** [comment]
Approved the workflow, lets see if this pass now.

**Comment by alexander-akait** [comment]
@TheRealSyler Hello, do you want to finish it?

**Comment by TheRealSyler** [comment]
@alexander-akait i forgot about this, i can take a look at it, next week or maybe the one after that.

**Comment by alexander-akait** [comment]
Thank you

**Comment by TheRealSyler** [comment]
@alexander-akait i looked at this again and it seems like the issue itself is already solved, i ran all the tests locally (one test failed but i don't think its relevant to this pr) and used my local version to bootstrap a sass project.

The issue seems to be that the ci fails, the logs of the ci output have already been deleted, you guys need to retrigger the ci to see why it fails, if i remember correctly the windows-latest tests take so long that they get aborted.

**Comment by snitin315** [comment]
Can you rebase your PR?

**Comment by TheRealSyler** [comment]
@snitin315 i never used rebase before but i think i did it.

**Comment by snitin315** [comment]
Yeah. Looks like something went wrong.

**Comment by TheRealSyler** [comment]
I crated a new pr (#3412)

**Pull request closed without merging**